playbooks/sqalab/run.yaml

```diff
@@ -0,0 +1,24 @@
+- hosts: all
+  tasks:
+    - name: 'scedule test run'
+      command: |
+        /snap/bin/weebl-tools.sqalab build add \
+          --deployment-branch {{ sqalab.branch }} \
+          --lab {{ sqalab.lab }} \
+          --format json \
+      register: build_info
+    - name: 'wait for test run'
+      command: /snap/bin/weebl-tools.sqalab build show --format json {{ (build_info.stdout | from_json)[0].uuid }}
+      register: build_info
+      until: (build_info.stdout | from_json)[0].status in ["Finished", "Canceled", "Aborted"]
+      delay: 600
+      retries: 90
+    - name: 'print info'
+      shell: |
+        /snap/bin/weebl-tools.sqalab build show {{ (build_info.stdout | from_json)[0].uuid }}
+        cat << EOF
+          Testrun results are summarized at:
+              https://solutions.qa.canonical.com/v2/testruns/{{ (build_info.stdout | from_json)[0].uuid }}
+          Generated artifacts are collected at:
+              https://oil-jenkins.canonical.com/artifacts/{{ (build_info.stdout | from_json)[0].uuid }}/index.html
+        EOF
```

Comment:
Let's discuss how we can improve the presentation of the results, this is just to have something in the meantime.